src/executor/source/routes/VersionRouter.ts

```diff
@@ -0,0 +1,16 @@
+import express from 'express'
+import { getApiVersion } from '../ApiInterface'
+import Logger from '../utils/logging/Logger'
+
+const VersionRouter = express.Router()
+
+VersionRouter.get( '', ( req, res ) => {
+  getApiVersion().then( ( apiResponse ) => {
+    res.status( 200 ).send( apiResponse )
+  } ).catch( ( reason ) => {
+    Logger.error( reason )
+    res.status( 500 ).send()
+  } )
+} )
```

Comment:
Can we use `async` and `await`?

Reply:
Sure. I used callback so I don't have to use a catch/try block, but I'm fine with either way of doing it